lib/core/elastic_connector_actions.rb

```diff
@@ -98,6 +105,7 @@ def complete_sync(connector_id, job_id, status)
         update_connector_fields(connector_id,
                                 :last_sync_status => sync_status,
                                 :last_sync_error => status[:error],
+                                :error => status[:error],
```

Comment:
Is `sync_error` supposed to be populated into `error`? It's already been set in `last_sync_error`

Reply:
Yeah I think this one should be `last_sync_error`, though I find having two errors in the index a bit confusing

Reply:
It's supposed to be that way, AFAIK. The [UI issue](https://github.com/elastic/enterprise-search-team/issues/2730) clearly states that the error message that will bubble up will come from the `error` field, and that field only. It won't be an aggregation of two.

As far as I understand, the `last_sync_error` is there just for the sync errors, but the `error` is for everything - not **everything else**. (Or at least that's how I understood the [Slack discussion](https://elastic.slack.com/archives/C03GAPTQ2JF/p1662736785742689?thread_ts=1662722221.419509&cid=C03GAPTQ2JF) - WDYT @sphilipse ?)